nextflow.config

```
@@ -214,6 +229,7 @@ profiles {
     apptainer {
         apptainer.enabled       = true
         apptainer.autoMounts    = true
+        if (params.use_gpu) { apptainer.runOptions = '--nv' }
```

This will make the Nextflow language server happy
```suggestion
        params.use_gpu ? '--nv' : apptainer.runOptions
```